agent/drive_handler_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package main
+
+import (
+	"testing"
+)
+
+func TestLsblk(t *testing.T) {
+	dh := driveHandler{}
+	err := dh.DiscoverDrives()
+	if err != nil {
+		t.Fatalf("E! %v", err)
+	}
+
+	t.Log(dh.drives)
```

Comment:
Are you planning to have a real test here?

Reply:
Yea, once I am able to get it working, Ill cleanup the code and add tests

Reply:
Yea, once I am able to get it working, Ill cleanup the code and add tests